### DISCUSSION TITLE
Unable to find any supported Python versions I'm aiming to deploy a python flask project

### DISCUSSION THREAD

**Discussion by tmitchmachine** [vercel/community#6326] [Help]
### Summary

WARN! Due to builds existing in your configuration file, the Build and Development Settings defined in your Project Settings will not apply. Learn More: https://vercel.link/unused-build-settings
Error: Unable to find any supported Python versions. The user is experiencing a 404 error when attempting to deploy their project to a live website. They are seeking assistance from a representative to resolve this issue.

### Steps to Reproduce

Inside of my repo https://github.com/tmitchmachine/mitch_deal
Unable to deploy it to a live website https://mitchdeal.com

Opening terminal in VS Code and type command vercel

**Comment by swarnava** [comment]
Can you go to Project Settings and make sure Node 18.x is selected?
